app/extensions.js

```diff
@@ -15,23 +17,51 @@ let generateBraveManifest = () => {
       {
         run_at: 'document_start',
         all_frames: true,
-        matches: ['<all_urls>'],
+        matches: ['http://*/*', 'https://*/*', 'file://*'],
```

Comment:
pretty sure they are needed at least for `data:` URLs because those can be HTML documents